src/baldr/edgeinfo.cc

```diff
@@ -96,6 +96,38 @@ std::vector<std::string> EdgeInfo::GetNames() const {
   return names;
 }
 
+// Get a list of names tagged and/or untagged with tag status
+std::vector<std::pair<std::string, bool>> EdgeInfo::GetNames(bool include_tagged_values) const {
+  // Get each name
+  std::vector<std::pair<std::string, bool>> name_type_pairs;
+  name_type_pairs.reserve(name_count());
+  const NameInfo* ni = name_info_list_;
+  for (uint32_t i = 0; i < name_count(); i++, ni++) {
+    // Skip any tagged names (FUTURE code may make use of them)
+    if (ni->tagged_ && !include_tagged_values) {
+      continue;
+    }
+    if (ni->tagged_) {
+      if (ni->name_offset_ < names_list_length_) {
+        std::string name = names_list_ + ni->name_offset_;
+        try {
```

Comment:
@kevinkreiser 
Just making sure that this is the try & catch you want me to remove, even that is similar to the one in line 189 bellow, which then leads me to ask why do you have it in the first place on GetNamesAndTypes.
This new function is like GetNamesAndTypes without filling up the types. (hence the new name I used)

Reply:
@kevinkreiser
Need feed back here

Reply:
yes this is the one im talking about. i dont know why its there at all! if i had to guess id say it had a purpose, the stuff inside the `try` got refactored and then it became vestigial. we should just remove both this one and the other

Reply:
done